zomes/blog/code/src/blog.rs

```diff
@@ -87,6 +87,13 @@ pub fn handle_my_posts() -> serde_json::Value {
     }
 }
 
+
+pub fn handle_my_posts_as_commited() -> serde_json::Value {
+    match hdk::query("post",0) {
```

Comment:
Maybe had a comment explaining what the 0 means?

Reply:
done